api_client/python/timesketch_api_client/client.py

```diff
@@ -22,6 +22,7 @@
 import bs4
 import requests
 # pylint: disable=redefined-builtin
+from altair.examples.multiline_tooltip import rules
```

Comment:
what is this??? 

and I don't see it ever being used, and I don't see why it should be used

Reply:
wow, no idea... removed ...